src/app/components/bitcoin-contract-entry-point/bitcoin-contract-entry-point-layout.tsx

```diff
@@ -1,5 +1,4 @@
 import { Flex, StackProps } from '@stacks/ui';
```

Comment:
Can you remove `@stacks/ui` here?

Reply:
Sure, I could, only thing is that the leather Flex lacks the 'as' property, which is quite important for me here, since I use the entry point Flex as a button. Do you have a solution to that? Thank you!

Reply:
Sure!